mobile_app/lib/view/cafe_detail/cafe_detail_section_title.dart

```diff
@@ -0,0 +1,26 @@
+import 'package:flutter/material.dart';
+import 'package:mobile_app/constants/color.dart';
+import 'package:mobile_app/view/common/badge.dart';
+
+class CafeDetailSectionTitle extends StatelessWidget {
+  final String title;
+  final bool? showBadge;
```

Comment:
```suggestion
  final bool showBadge;
```
constructor에 default value가 있어서 괜찮음
아래쪽에서 `!` type assertion도 삭제 부탁드립니다